app/theme/breakpoints.ts

```diff
@@ -0,0 +1,10 @@
+const breakpoints = {
```

Comment:
Maybe I ask you why you decided to add these specific breakpoints and what is wrong with Chakra's default breakpoints?

Reply:
The `base` and `small` has a `30em` difference. I noticed that the difference was too big, and I need a breakpoint that is in between them. I added the `xs` just for that